packages/@react-spectrum/labeledvalue/docs/LabeledValue.mdx

```diff
@@ -0,0 +1,151 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/labeledvalue';
+import types from 'docs:./types.ts';
+import {HeaderInfo, PropTable, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/labeledvalue/package.json';
+
+```jsx import
+import {LabeledValue} from '@react-spectrum/labeledvalue';
+```
+
+---
+category: Status
+keywords: [label, value, read only]
+after_version: 3.0.0
+---
+
+# LabeledValue
+
+<PageDescription>{docs.exports.LabeledValue.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['LabeledValue']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/field-label/'}
```

Comment:
Close enough for now?

Reply:
can we leave it empty for now? or maybe link to a readonly field at the moment? https://spectrum.adobe.com/page/text-field/#Read-only

I think I like leaving it off for now though. Anything else we link to will give the wrong idea of when to use this and what for